internal/librariangen/release/release_test.go

```diff
@@ -148,15 +152,38 @@ func TestInit(t *testing.T) {
 		},
 		{
 			name:        "changelog already up-to-date",
-			requestJSON: `{ "libraries": [ { "id": "secretmanager", "version": "1.16.0", "release_triggered": true, "apis": [{"path": "google/cloud/secretmanager/v1"}], "changes": [{"type": "feat", "subject": "add new GetSecret API"}] } ] }`,
+			requestJSON: `{ "libraries": [ { "id": "secretmanager", "version": "1.16.0", "release_triggered": true, "apis": [{"path": "google/cloud/secretmanager/v1"}], "changes": [{"type": "feat", "subject": "add new GetSecret API"}] } ], "tag_format": "{id}/v{version}" }`,
 			initialRepoContent: map[string]string{
 				"secretmanager/CHANGES.md": "# Changes\n\n## [1.16.0](https://github.com/googleapis/google-cloud-go/releases/tag/secretmanager%2Fv1.16.0)\n- Already there.",
 				"internal/generated/snippets/secretmanager/apiv1/snippet_metadata.google.cloud.secretmanager.v1.json": `{"version": "1.15.0"}`,
 			},
+			moduleRootPath:           "secretmanager",
 			changelogAlreadyUpToDate: true,
 			wantVersion:              "1.16.0",
 			wantSnippetVersion:       `"version": "1.16.0"`,
 		},
+		{
+			name: "whole repo library",
+			requestJSON: `{
+				"libraries": [{
+					"id": "wholerepo", "version": "1.16.0", "release_triggered": true,
+					"source_roots": ["."],
```

Comment:
Can you expand this example (or add another) that involves multiple source_roots entries, as I believe google-cloud-go will have? That would also be a good way to lock down the `slices.Contains` logic in `wholeRepoLibrary`.

Reply:
I'm not expecting to have any repos that have "." *and* something else. For the google-cloud-go "main" module, I'm expecting to have source roots of "civil", "rpcreplay", "httpreplay" and some "internal" entries. I haven't figured out how we still write to the root CHANGES.md file in that situation though. (I don't think we'd want "." as so many other libraries would need to be excluded from the releases...)